applications/nrf_desktop/src/modules/Kconfig.dvfs

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DVFS_ENABLE
```

Comment:
`_ENABLE` is not needed.
As far as I see newer Kconfig options in nRF Desktop do not use it.

Reply:
Both approaches have some pros and cons (this one explicitly shows that option enables a module). I would not enforce selecting one of the approaches.

Reply:
IMO adding `_ENABLE` seems repetitive as by writing `=y` we are already saying that we are enabling the option, that is why i suggested it.

Reply:
The `_ENABLE` suffix explicitly means that the option is used to enable a new functionality and not e.g. to configure an existing functionality (so it's not repetitive). Still, I would not insist to keep it.